eks/cluster-loader/clusterloader2/cluster-loader.gotmpl

```diff
@@ -42,21 +42,34 @@ apiVersion: batch/v1
 kind: Job
 metadata:
   name: clusterloader2
-  namespace: clusterloader2
+  namespace: cl2
 spec:
   template:
     spec:
       containers:
-      - name: clusterloader2
-        image: {{ .Image }}
-        command: ["/clusterloader"]
-        args: {{ .TestArgs }}
-        volumeMounts:
-        - name: config
-          mountPath: /etc/config
+        - name: clusterloader2
+          image: {{ .Image }}
+          command: ["/clusterloader"]
+          args: {{ .TestArgs }}
+          volumeMounts:
+            - name: config
+              mountPath: /etc/config
+            - name: reports
+              mountPath: /usr/share
+              readOnly: false
+        - name: results
+          image: amazon/aws-cli:latest
+          command: ["/bin/bash"]
```

Comment:
If possible, I'd use "/bin/sh" here and remove any Bash-isms ... Bash is a pretty complex beast and I think for these kind of automation situation's it's better to stay vanilla

Reply:
Good idea. Will do.